react-components/src/components/RevealToolbar/LayersContainer/elements.ts

```diff
@@ -0,0 +1,50 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { Checkbox, Chip, Menu } from '@cognite/cogs.js';
+import styled from 'styled-components';
+
+export const StyledCheckbox = styled(Checkbox)`
+  margin-left: 10px;
+`;
+
+export const StyledMenu = styled(Menu)`
+  padding: 6px;
+`;
+
+export const StyledSubMenu = styled(Menu)`
+  box-shadow: none;
+  padding: 8px;
+`;
+
+export const StyledChipCount = styled(Chip)`
+  && {
+    background: #5874ff;
+    border-radius: 2px;
+    width: fit-content;
+    height: 20px;
+    max-height: 20px;
+    min-height: 20px;
+    min-width: 20px;
+    padding: 4px;
+    color: #ffffff;
+
+    /* Font */
+    font-family: 'Inter';
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+  }
```

Comment:
```suggestion
  && {
    border-radius: 2px;
    width: fit-content;
    height: 20px;
    max-height: 20px;
    min-height: 20px;
    min-width: 20px;
    padding: 4px;
  }
```

You don't need any change of color since `neutral` type of `Chip` component already includes them.

Reply:
Also, these colors are not following the colors from the design. 
Here is how it looks without changes:
<img width="221" alt="Screenshot 2023-07-21 at 14 08 08" src="https://github.com/cognitedata/reveal/assets/45129444/84f340fb-b313-4550-80a5-a3c11d83d326">

Here is the design:
<img width="501" alt="Screenshot 2023-07-21 at 14 06 33" src="https://github.com/cognitedata/reveal/assets/45129444/d753b304-0571-40db-b373-1c2192d1ef40">
And here is it after changes with setting `type` prop on `Chip` component to `neutral`:
<img width="221" alt="Screenshot 2023-07-21 at 14 07 46" src="https://github.com/cognitedata/reveal/assets/45129444/615361fb-9d37-4648-9a71-69ccf93ffd89">